lib/WP_Auth0_LoginManager.php

```diff
@@ -321,8 +326,15 @@ public function implicit_login() {
 			);
 		}
 
-		// Populate legacy userinfo property.
-		$decoded_token->user_id = $decoded_token->sub;
+		// Remove unneeded ID token attributes.
+		foreach ( array( 'iss', 'aud', 'iat', 'exp', 'nonce', 'clientID' ) as $attr ) {
```

Comment:
what's the reason to remove them? how will you know if a token works for a given audience or if the token is still valid (without making a network request)?

Reply:
This is just saving userinfo, we don't use it as a token. I'm trying to get the information that we're saving to be more consistent between the two login flows. Also, this isn't saved as `id_token` so it's a bit confusing to use it as such.